pkg/ddc/thin/transform_fuse.go

```diff
@@ -143,6 +149,70 @@ func (t *ThinEngine) parseFuseImage(runtime *datav1alpha1.ThinRuntime, value *Th
 	}
 }
 
+func (t *ThinEngine) parseHostVolumeFromDataset(dataset *datav1alpha1.Dataset, thinValue *ThinValue) error {
+	index := 0
+	for _, mount := range dataset.Spec.Mounts {
+		for key, value := range mount.Options {
+			if key != common.DatasetOptionFluidFuseHostVolume {
+				continue
+			}
+
+			var hostPath, mountPath string
+			subStrings := strings.Split(value, ":")
+			if len(subStrings) > 2 {
+				return fmt.Errorf("invalid dataset option format, %s: %s", key, value)
+			}
```

Comment:
"invalid dataset option format, %s: %s" uses "format." while 
"invalid dataset option, %s: %s, should be a absolute hostPath" does not mention "format." I suggest using `dataset option`.